server/atelier-queries.js

```diff
@@ -54,6 +54,14 @@ const getProductQuestions = (productId, headers) => {
     });
 };
 
+const postInteraction = (interaction, headers) => {
+  return axios.post('https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/interactions', interaction, headers)
+    .then(result => result.data)
+    .catch(err => {
+      throw err;
```

Comment:
I know we have eschewed console.logs for catch blocks, is there a plan for top-most error-handling from various throw statements?

Reply:
This is an important question. We should figure out error handling and all that as a group